src/ledger/backend/contracts_deposit.ts

```diff
@@ -3,16 +3,16 @@
 
 import { ethers, Signer, BigNumber } from "ethers";
 
-import { ETHHolder__factory } from "./contracts/factories/ETHHolder__factory";
-import { IERC20__factory } from "./contracts/factories/IERC20__factory";
-import { IERC721__factory } from "./contracts/factories/IERC721__factory";
-import { ERC20Holder__factory } from "./contracts/factories/ERC20Holder__factory";
-import { ERC721Holder__factory } from "./contracts/factories/ERC721Holder__factory";
-import { Address } from "../";
-import { Asset } from "../assets";
-import { Amount } from "../assets";
-import { Tokens } from "../assets";
-import { DepositCalls } from "./tokenmanager";
+import { ETHHolder__factory } from "#erdstall/ledger/backend/contracts/factories/ETHHolder__factory";
+import { IERC20__factory } from "#erdstall/ledger/backend/contracts/factories/IERC20__factory";
+import { IERC721__factory } from "#erdstall/ledger/backend/contracts/factories/IERC721__factory";
+import { ERC20Holder__factory } from "#erdstall/ledger/backend/contracts/factories/ERC20Holder__factory";
+import { ERC721Holder__factory } from "#erdstall/ledger/backend/contracts/factories/ERC721Holder__factory";
```

Comment:
this is a special case... if the sub-dir structure `./contracts/factories/*` is tightly coupled to this directory `backend`, I'd also keep local imports, because then the dir-tree `backend/**` is then one coherent module. If something is not coupled, I'd use the `#erdstall` anchor.